### PR TITLE
Phasetwo

### PR DESCRIPTION
**Pull request by upexcel**

### CODE REVIEW

File: src/app/modules/edit-variable/edit-variable.component.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<div class="header">
+    <p class="h-text">Edit Variable</p>
+</div>
+<form autoComplete="off" #addTemp="ngForm" (ngSubmit)="update(addTemp)">
+    <div class="con-pg" fxLayout fxLayout.xs="column" fxLayoutAlign="center" fxLayoutGap="10px" fxLayoutGap.xs="0">
+        <div class="main" fxFlex="75%">
```
Comment (excellencetechnologies) [inline src/app/modules/edit-variable/edit-variable.component.html]: can we have same html for add/edit

File: src/app/modules/inbox/inbox.component.ts [hunk 2]
```diff
@@ -72,7 +73,7 @@ export class InboxComponent implements OnInit {
 
     reject() {
         this.selected = {
-            'tag_id': '1',
+            'tag_id': '2',
```
Comment (excellencetechnologies) [inline src/app/modules/inbox/inbox.component.ts]: use constants instead of 1,2

File: src/app/service/mock-data.ts [hunk 3]
```diff
@@ -94,17 +48,17 @@ export const UserVariable: any = [{
     {
         '_id': '4394sdfdfj',
         'variableCode': '#director_signature',
```
Comment (excellencetechnologies) [inline src/app/service/mock-data.ts]: remove this file

File: src/app/modules/inbox/inbox.component.ts (hunk 2 above)
Comment (upexcel) [inline src/app/modules/inbox/inbox.component.ts]: getting error "A class member cannot have the 'const' keyword". so i have used readonly

File: src/app/service/mock-data.ts (hunk 3 above)
Comment (upexcel) [inline src/app/service/mock-data.ts]: removed all unused mock data. i am still using system variable from this file. once i get system variable api. i will remove  this also.

**Pull request merged**